asciidoc/courses/graph-data-science-fundamentals/modules/1-graph-algorithms/lessons/2-centrality-and-importance/lesson.adoc

```diff
@@ -46,23 +57,28 @@ In summary, PageRank estimates the importance of a node by counting the number o
 
 Below is an example of applying PageRank to find the most influential persons in the Director -> Actor network from movies released on or after 1990 with a revenue of at least 10 Million dollars.
 
-First, create the graph projection. We can use a Cypher projection in this case to obtain a graph where we have `(Person)-[:DIRECTED_ACTOR]->(Person)`. this graph can be traversed to understand the influence across directors and actors.
+First, create the graph projection. We can use a Cypher projection in this case to obtain a graph where we have `(Person)-[:DIRECTED_ACTOR]->(Person)`. This graph can be traversed to understand the influence across directors and actors.
```

Comment:
I think it would make sense to briefly explain what a projection is.  I don't think this is covered is the course yet

```
First, create the graph projection. We can use a Cypher projection to create an in-memory graph with `:DIRECTED_ACTOR` relationships between two `(:Person)` nodes. This graph can be traversed to understand the influence across directors and actors.
```

Reply:
I have included a brief description of a graph projection and a link to Introduction to Neo4j Graph Data Science